app.json

```diff
@@ -1,8 +1,8 @@
 {
   "expo": {
-    "name": "discovery-mobile-ui",
-    "slug": "discovery-mobile-ui",
-    "owner": "sync-for-science",
+    "name": "discovery-mobile-ui-karanl1",
+    "slug": "discovery-mobile-ui-karanl1",
+    "owner": "discovery-mobile-ui-karan",
     "version": "1.0.0",
```

Comment:
I think this version may need to be bumped.

Reply:
resolved